tests/e2e/util.go

```diff
@@ -3619,7 +3680,13 @@ func sshExec(sshClientConfig *ssh.ClientConfig, host string, cmd string) (fssh.R
 func createPod(client clientset.Interface, namespace string, nodeSelector map[string]string,
 	pvclaims []*v1.PersistentVolumeClaim, isPrivileged bool, command string) (*v1.Pod, error) {
 	pod := fpod.MakePod(namespace, nodeSelector, pvclaims, isPrivileged, command)
-	pod.Spec.Containers[0].Image = busyBoxImageOnGcr
+	commands := []string{"Powershell.exe", "-Command", command}
```

Comment:
you can move this portion also to the constant

Reply:
the command is a parameter in commands var, so I have kept in this method.